server/classes/SSR.js

```diff
@@ -0,0 +1,56 @@
+import axios from 'axios';
+import React from 'react';
+import ReactDOMServer from 'react-dom/server';
+import ReviewService from '../../services/reviewService/components/ReviewsContainer';
+
+class SSR {
+    constructor(baseUrl, html) {
+        this.baseUrl = baseUrl;
+        this.html = html;
+        this.reviewsData = null;
+    }
+
+    getId() {
+        const splitUrl = this.baseUrl.split('/');
+        return splitUrl[splitUrl.length - 1];
+    }
+
+    async getReviewsData() {
+        const { data } = await axios.get(`${process.env.REVIEWS_DOMAIN}/api/reviews/all/${this.getId()}`);
```

Comment:
I notice you're using process.env here but not using the dotenv package. Is this linked up with your docker image?

Reply:
Yes, I set my ENVs in the docker containers, but if I run this locally, I use a plugin of my Oh-My-Zsh terminal that automatically puts variables in .env files into my environment. Super useful tool!